### PR TITLE
tree: rename devlib -> digga

### PR DESCRIPTION
**Pull request by Pacman99**

### CODE REVIEW

File: src/modules.nix [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
       _module.args = {
         inherit self;
-        devlib = pkgs.devlib;
+        diggaLib = pkgs.lib.digga;
```
Comment (blaggacao) [inline src/modules.nix:48]: `# diggaLib can be accessed in modules directly as diggaLib`

**Review by blaggacao — APPROVED** [review]
I liked the friendly comment on the other site so much, I'd like to see it here as well. :smile:

File: src/modules.nix (hunk 1 above)
Comment (Pacman99) [reply src/modules.nix:49]: sounds good

Comment (Pacman99) [reply src/modules.nix:49]: changed it to:
`# digga lib can be accessed in modules directly as diggaLib`
thats more accurate

**Pull request merged**